controllers/authController.js

```diff
@@ -0,0 +1,142 @@
+
+
+const User = require("./../models/User.model")
+const bcryptjs = require("bcryptjs")
+
+exports.viewRegister = (req, res) => {
+
+    res.render("auth/signup")
+
+}
+
+exports.register = async (req,res)=> {
+
+//1. OBTENCION DE DATOS DEL FORMULARIO
+const username = req.body.username
```

Comment:
Con la siguente destructuracion podemos acceder a las propiedas y sus valores para obtenerlas como variables aisladas y manipularlas, esto nos ahorra lineas de codigo y es muy util cuando tenemos varias propiedades 😉 
```suggestion
const {username, password} = req.body
```